guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>Once the explanatory variables and explanatory rasters are defined, we <s>could</s> <u>can</u> prepare our TabularDataObject by using <code>prepare_tabulardata</code> function. </p><p><code>input_features</code>: It takes in data in the form of feature layer<u>s</u> or spatially <s>enables</s> <u>enabled</u> dataframe<u>s</u>. Feature layers <s>could</s> <u>can</u> be of point or polygon geometry. </p><p><code>variable_predict</code>: Field name from the <code>input_features</code> layer. It automatically deduces fields as continuous or categorical <s>based</s> <u>by</u> using a simple logic. If the field type <s>if</s> <u>is</u> float and <s>it</s> has more than 20 unique values, the field is considered continuous. <u>O</u>therwise, <u>the field will be considered</u> categorical. </p><p><code>date_field</code>: Field name from the <code>input_features</code> layer. If specified, the field will be split into year, month, week, day, dayofweek, dayofyear,  is_month_end, is_month_start, is_quarter_end,  is_quarter_start, is_year_end, is_year_start,  hour, minute, second, elapsed.  If specified here, <u>there is </u>no need to specify as part of <code>explanatory_variables</code>. </p><p><code>explanatory_variables</code> : List of the field names as 2-sized tuples containing the explanatory variables from the <code>input_layer</code> </p><p><code>explanatory_rasters</code> : List of the field names as 2-sized tuples containing the explanatory variables in multiple input rasters </p><p>distance_features: Distance features are used to automatically create explanatory variables by calculating a distance from the provided features to the input_features. Distances will be calculated from each of the input explanatory distance features to the nearest Input training features. If the input <code>distance_features</code> are polygons or lines, the distance attributes are calculated as the distance between the closest segments of the pair of features. However, distances are calculated differently for polygons and lines. See how proximity tools calculate distance for details.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>